code/modules/organs/subtypes/nymph_limbs.dm

```diff
@@ -237,11 +237,11 @@
 		H.client && H.stat == CONSCIOUS)
 			mob_list += H
 
-	if(!LAZYLEN(GLOB.mob_list))
+	if(!LAZYLEN(mob_list))
```

Comment:
may be worth renaming this var to prevent confusion in the future, but not a blocker